sourcecode/hub/production.Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM laravelphp/vapor:php82
+
+COPY . /var/task
```

Comment:
Each line is supposed to be terminated with a line feed, according to the rules in `.editorconfig`. This doesn't make much difference for YAML files, but it matters for e.g. shell scripts, so might want to ensure your editor abides by `.editorconfig`.

Reply:
These files were added by Vapor during initialization, which is why they didn't abide by `.editorconfig`.
I've added line feed terminations to both this file and `staging.Dockerfile`.